src/dlite-entity.c

```diff
@@ -2415,7 +2415,8 @@ const char *dlite_metamodel_missing_value(const DLiteMetaModel *model)
   for (i=0; i < model->meta->_nproperties; i++) {
     DLiteProperty *p = model->meta->_properties + i;
     if (strcmp(p->name, "dimensions") == 0) {
-      if (!model->dims) return p->name;
+      //if (!model->dims) return p->name;
```

Comment:
If I understand it right, you commented this out to allow instances with no dimensions. 
That sounds right. If so, remove the comment to keep the code clean.